app/_locales/zh_TW/messages.json

```diff
@@ -766,19 +902,19 @@
     "message": "重置帳戶將清除您的交易紀錄"
   },
```

Comment:
```suggestion
  },
  "restore": {
    "message": "恢復"
  },
```

Reply:
fixed in https://github.com/Alpha-Carbon/wallet-extension/pull/21/commits/47dd284d4cacbca212317a5b0b499afa6a996e36